pkg/gateway/vald/handler/grpc/handler.go

```diff
@@ -130,7 +130,8 @@ func (s *server) search(ctx context.Context, cfg *payload.Search_Config,
 			span.End()
 		}
 	}()
-	maxDist := uint32(math.MaxUint32)
+	var maxDist uint32
+	atomic.StoreUint32(&maxDist, math.Float32bits(math.MaxFloat32))
```

Comment:
I think `float32(math.Inf(0))` is more precise representation than `math.MaxFloat32`, but it makes little difference in practical.
How about it?

Reply:
math.Inf returns float64 we can't use here